src/Apps/Collect2/Components/SeoProductsForArtworks.tsx

```diff
@@ -51,14 +51,17 @@ export class SeoProducts extends React.Component<SeoProductsProps> {
           ? toSentence(artists.map(artist => artist!.name))
           : null
         const isInstitution = partner && partner.type === "Institution"
+        const partnerImg = get(partner, p => {
```

Comment:
so with `get` you don't need to be defensive with the `&&` stuff, that's why it can be kind of cleaner for this sort of thing.

can just be:

```
const partnerImage = get(partner, p => p.profile.icon.url)
```